src/task/SetLocalDescriptionTask.ts

```diff
@@ -33,6 +33,12 @@ export default class SetLocalDescriptionTask extends BaseTask {
     const sdpOfferInit = this.context.sdpOfferInit;
     let sdp = sdpOfferInit.sdp;
 
+    if (this.context.browserBehavior.hasChromiumWebRTC()) {
```

Comment:
nit: We generally try to add new API that is specific to the browser behavior (in this case something like supportsVideo...HeaderExtension).

Reply:
I think technically (maybe a bit overkill) the more future prepped (i.e. in case FF/others decide to add support/support munging) check would be [this.meetingSessionContext.transceiverController.localVideoTransceiver().sender.getParameters](https://github.com/aws/amazon-chime-sdk-js/blob/8e7c0402c89837fe2dd623075003bbf679d28247/src/audiovideocontroller/DefaultAudioVideoController.ts#L964) . Since we are checking with that to see whether it ends up being negotiated. Limiting to just chrome is fine though (the helper function as trung mentioned would be nice as well).

Reply:
Not sure what is the desired check here. Are we sticking to limiting to chrome? @ltrung what do you think?

Reply:
Let's just limit to chrome for now, but we can abstract that choice (and technically allow builder override) by adding a function like `supportsVideoLayersHeaderExtensionMunging` to https://github.com/aws/amazon-chime-sdk-js/blob/main/src/browserbehavior/DefaultBrowserBehavior.ts .

I was just mentioning that we could theoretically make the check broader, as long as the browser supports `getParameters` we can check whether the extension was negotiated or not. Not important though.

Reply:
Okay, added a function `supportsVideoLayersAllocationRtpHeaderExtension`.